app/models/post.rb

```diff
@@ -73,7 +73,7 @@ def self.by_slug(slug, user)
       return nil
     end
 
-    if post&.help_category == '$Moderator' && !user&.is_moderator
+    if post&.help_category == '$Moderator' && !user&.at_least_moderator?
```

Comment:
Untested suggestion:

```suggestion
    if post&.help_category == '$Moderator' && user&.standard?
```

Reply:
Unresolving this - reverted for now, for some reason I am not sure yet, it causes a test failure despite the `standard?` method being just a negation of `at_least_moderator?`

Reply:
Let's not apply these ones as the actual switch is `!user || user.standard?` which is not what you hoped for, I believe

Reply:
Oh I see. Well spotted. Yes that's more work to read than the original. There are a few suggested changes that don't involve the safe navigation operator, which I'm guessing would still work? I don't know what's best for consistency though.